### PR TITLE
Updated success and error text

### PR DESCRIPTION
**Pull request by RohitKandimalla**
## Checklist

- [ ] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [ ] Tests are included and test edge cases.
- [ ] Tests have been run locally and pass.

## Screenshots (if appropriate)

- [ ] None applicable

### CODE REVIEW

File: src/main/java/mat/client/export/measure/ManageMeasureExportPresenter.java [hunk 1]
```diff
@@ -88,9 +88,9 @@ public void onFailure(Throwable throwable) {
 			public void onSuccess(Boolean success) {
 				Mat.hideLoadingMessage();
 				if (success) {
-					view.displaySuccessMessage("Measure is being processed and transferred to MADIE, You will receive an email as soon as transfer is completed");
+					view.displaySuccessMessage("Measure is being processed and transferred to MADiE, You will receive an email as soon as transfer is completed");
```
Comment (jkotanchik-SB) [inline src/main/java/mat/client/export/measure/ManageMeasureExportPresenter.java:91]:
```suggestion
					view.displaySuccessMessage("Measure is being processed and transferred to MADiE. A message will be spent to the e-mail associated with this account once the transfer has completed.");
```

File: src/main/java/mat/client/export/measure/ManageMeasureExportPresenter.java [hunk 2]
```diff
@@ -88,9 +88,9 @@ public void onFailure(Throwable throwable) {
 			public void onSuccess(Boolean success) {
 				Mat.hideLoadingMessage();
 				if (success) {
-					view.displaySuccessMessage("Measure is being processed and transferred to MADIE, You will receive an email as soon as transfer is completed");
+					view.displaySuccessMessage("Measure is being processed and transferred to MADiE. A message will be spent to the e-mail associated with this account once the transfer has completed.");
```
Comment (jkotanchik-SB) [inline src/main/java/mat/client/export/measure/ManageMeasureExportPresenter.java:91]:
```suggestion
					view.displaySuccessMessage("Measure is being processed and transferred to MADiE. A message will be sent to the e-mail associated with this account once the transfer has completed.");
```

My bad. Suggested `spent` instead of `sent`.

**Review by jkotanchik-SB — APPROVED** [review]
